packages/eas-cli/src/utils/download.ts

```diff
@@ -0,0 +1,151 @@
+import spawnAsync from '@expo/spawn-async';
+import cliProgress from 'cli-progress';
+import glob from 'fast-glob';
+import fs from 'fs';
+import fetch, { RequestInit } from 'node-fetch';
+import path from 'path';
+import { Stream } from 'stream';
+import { extract } from 'tar';
+import tempy from 'tempy';
```

Comment:
I would prefer to avoid tempy, in other places we use sth like `path.join(getTmpDirectory(), uuidv4())`.